app/src/server/services/email/charge.ts

```diff
@@ -840,16 +840,20 @@ export const sendChargeTemplate = async () => {
     .catch(error => console.error(error));
 }
 
-export const sendEmail = async (receiverEmail: string, fields: {chargerUsername: string, homeName: string, amountOwed: string, dueDate: Date, comment: string}) => {
+export const sendEmail = async (receiverEmail: string, fields: {chargerUsername: string, homeName: string, amountOwed: string, dueDate: Date, address: string}) => {
```

Comment:
Since paymentMethods are also on the charge user it would be nice to have the paymentMethod links passed in and can just be iterated out to list bullets.